lib/hint.rb

```diff
@@ -16,34 +16,43 @@
 # you may find current contact information at www.suse.com
 
 class Hint
-  def self.get_started
-    output "You can get started by inspecting a system. Run:\n#{$0} inspect HOSTNAME"
+  def self.print(method, options = {})
+    if Machinery::Config.new.hints
+      Machinery::Ui.puts to_string(method, options)
+    end
+  end
+
+  def self.to_string(method, options = {})
+    Machinery::Config.new.hints ? "\nHint: #{send(method, options)}\n" : ""
+  end
+
+  private
+
+  def self.get_started(options)
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `get_started(*)` if you want the method to accept any arguments but don't care about them.